package.json

```diff
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap --hoist",
+    "bootstrap": "lerna bootstrap --concurrency=1 --mutex network --hoist",
```

Comment:
What are these extra options for?

Reply:
Oh! sorry, that should not be there. I had to add them for a windows specific thing that was solved with admin permissions at the end, will remove them and commit again